README.md

```diff
@@ -16,16 +16,26 @@ To report an issue using the **Language Forge** service, email "languageforgeiss
 
 ## Developers ##
 
-We use [Gitflow](http://nvie.com/posts/a-successful-git-branching-model/) mostly, however we do not utilize a `develop` branch.  All work starts and returns to the `master` branch.  Deployment of software to a staging and/or a production environment occur through the use of the respective branches being fast-forwarded to the appropriate commit on `master` which will in turn kick off our automated deployment processes.
-
-| Master Branch | Staging Branch (for testing) | Production Branch |
-| ------------- | --------- | ----------- |
-| `master` | `staging` | `prod` |
+We use a modified [Gitflow](http://nvie.com/posts/a-successful-git-branching-model/) process to manage changes.
+
+### Typical ###
+1.  Create a branch off of `develop`
+1.  Create a PR back into `develop`
+1.  Once approved and merged, test those changes in the staging environment
+1.  Once you are satisifed with the changes, open another PR back into `master`
+1.  Once approved and merged, coordinate with the team to deliver those changes to the production environment
+1.  Team leads will determine when the right time to cut a release, i.e., tag and deploy, typically this should be within a day or two
+
+### Exception ###
+1.  There may be times when it's necessary to create a branch off of `master`
+1.  A PR should be created to `master`
+1.  Once approved and merged, coordinate with the team to deliver those changes to the production environment
+1.  Team leads will determine when the right time to cut a release, i.e., tag and deploy, this would likely be right away under these exceptional circumstances
```

Comment:
I don't require a PR to merge develop into master, just as I didn't require a PR to merge staging into master.  There should be no unreviewed code in these merges, and my opinion is that a PR will be unnecessary friction that will slow down deployment, especially give the time zone difference.  @longrunningprocess, @rmunn and myself are authorized to merge into master and deploy as desired.  When merging to master and/or deploying, an announcement should be made on Slack to make all parties aware.

Reply:
ok, let's talk through the handoff between the dev and one of us to get it merged into `master` then, without a PR, will we simply rely on the dev to let us know they've finished testing it?

Reply:
I'll make the changes and we can discuss at tonight's stand-up